doc/source/installing.rst

```diff
@@ -407,7 +407,7 @@ development version of yt instead of the latest stable release, you will need
   $ git clone https://github.com/yt-project/yt
   $ cd yt
   $ git checkout master
-  $ pip install -e . --user --prefix=
+  $ pip install . --user --prefix=
```

Comment:
And please also update the notes about why we're using `--user` and `--prefix` in this document.